pkg/api/annotations/pod.go

```diff
@@ -27,6 +27,14 @@ const PodLaunchType = "pod.elotl.co/launch-type"
 // override specified resource requests and limits.
 const PodInstanceType = "pod.elotl.co/instance-type"
 
+// PodImageOverlayRootfs is an annotation users add to their
+// kubernetes pods to inform kip whether the itzo cell will use
+// -mount or -extractto argument when itzo calls tosi to pull rootfs
+// images. This annotation will default to -mount if its value is empty
```

Comment:
Might want to add a sentence or modify the sentence to say what each of those arguments does and how the behavior of the system is different between the two.

Reply:
I'd change this comment to something like:

    // PodImageOverlayRootfs is an annotation users add to their
    // kubernetes pods to inform kip whether to use overlayfs when
    // setting up the container images, or a directory with all the image layers
    // extracted. The default is overlayfs. Set the label value to a truthy value
    // for overlayfs, and to a falsy value to have image layers extracted for the
    // rootfs without overlayfs.

What parameters are used under the hood is not important for the user, and it's an implementation detail.